### PR TITLE
Tizen 5.5 Public M3 Release Note

### PR DESCRIPTION
**Pull request by wtprince**
Initial version of Tizen 5.5 Public M3 Release Note

### CODE REVIEW

**Comment by journey2w** [comment]
@annie-samsung 
Please review this PR.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:6]: The Tizen 5.5 Public release provides new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.

@mijin85cho, should the lead line be standard?

Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:6]: Release Details needs to be added.

Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:6]: The Tizen 5.5 Public M3 release provides new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.

@mijin85cho, @journey2w  should the lead line be standard like other release notes?

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:8]: Need to add Release Details, see previous release notes.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:9]: There needs to be heading 
##Release Notes 
and other headings should be adjusted accordingly, see previous release notes

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:93]: Are there any known issues for this release, that needs to be documented?

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:74]: colon at the end
 Following API set has been changed to provide custom time period support and return usage in mAh:

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:45]: Can it be rephrased as?
A new API for registering any change in sticker info has been added.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [inline docs/platform/release-notes/tizen-5-5-m3.md:55]: int SendData(byte[] data) has been added.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:9]: We haven't used "##Release Notes" before. We used the same current headings for other headings. I think someone added this heading(##Release Notes) regardless of the result of the previous review request. Anyway, do we need to add "##Release Notes" and adjust other headings from this release?

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:8]: We haven't used "## Release Details" before. I think someone added this heading(## Release Details) regardless of the result of the previous review request.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:93]: No known issue for this release. This release is just for adding some features for wearable only.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:45]: will accept

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:55]: will accept for 4 lines.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:1]: OK. I will update it too.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:6]: @mijin85cho  agree , I do not prefer this as a list, missed to mention it explicitly.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:8]: @wtprince, @mijin85cho 

Thanks for clarifying.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:9]: @mijin85cho, could you confirm what needs to be done in this case?

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:74]: @mijin85cho agree with this

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (mijin85cho) [reply docs/platform/release-notes/tizen-5-5-m3.md:9]: For the release note, after the "release note" parts were merged, we have been added "Release Details" part separately.

If @wtprince add the "##Release Notes" title and adjust the title depth, it would be helpful for later work indeed. As for now we don't have "Release Details" information, I think modifying "##Release Notes" part only will be helpful enough. 
@wtprince would you?

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:6]: will accept

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:9]: OK. I will.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:74]: will accept

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:57]: will accept

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (wtprince) [reply docs/platform/release-notes/tizen-5-5-m3.md:38]: will accept

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (mijin85cho) [reply docs/platform/release-notes/tizen-5-5-m3.md]: > OK. I will.

Thank you for that.

File: docs/platform/release-notes/tizen-5-5-m3.md
Comment (mijin85cho) [reply docs/platform/release-notes/tizen-5-5-m3.md:58]: Thanks for the suggestion, @annie-samsung 
Thanks for the update, @wtprince.

However, I want to get your opinion about these 4 lines.

In the previous comment, we just changed these lines to get the sentence form.
However, as you can see in other sections, we usually write in below form.

API for sanding data in byte has been added.
API for sanding data in string has been added.
API to get data in byte has been added
API to get data in string has been deprecated.

(These sentences are based on my guess.)
So I'd like to suggest to change the sentences to get similar form with other sections.

Would you please let me know your opinion about my suggestion?

Comment (annie-samsung) [reply docs/platform/release-notes/tizen-5-5-m3.md:58]: @mijin85cho, agreed it is better to keep it uniform. That also gives better clarity on the purpose of the API. Recommend to check with developer for purpose of API and modify accordingly.

Comment (mijin85cho) [reply docs/platform/release-notes/tizen-5-5-m3.md:58]: @wtprince Would you please check if the sentences are accurate to the original meaning, and apply them?

Comment (semun-lee) [reply docs/platform/release-notes/tizen-5-5-m3.md:58]: @mijin85cho , the sentence has correct meaning.

API for **sanding** data in byte has been added. 

API for **sending** data in byte has been added.

**Comment by mijin85cho** [comment]
Thanks for the update, @wtprince 

@annie-samsung Would you please check the latest version, and approve it if you see no more issues?

FYI : As the release date is August 31, I'll hold this PR till that date even after this PR is aproved, for the case of any further update.

**Review by annie-samsung — APPROVED** [review]
Looks good to me, approving.

**Review by annie-samsung — APPROVED** [review]
Approving.

**Review by mijin85cho — APPROVED** [review]
Thanks for the updates, and checking.

**Pull request merged**
